pages/linux/mhwd.md

```diff
@@ -0,0 +1,28 @@
+# mhwd
+
+> Manjaro Hardware Detection utility  .
+> More information: <https://wiki.manjaro.org/index.php/Manjaro_Hardware_Detection_Overview>.
+
+- [l]ist available drivers:
```

Comment:
```suggestion
- List available drivers:
```
These are not needed. The short option is clearly short for the long option. Fix the rest of the page too.